src/style.css

```diff
@@ -58,19 +62,17 @@ nav {
   position: relative;
   background: url(Img/illustration-header-mobile.png);
   background-repeat: no-repeat;
-
-  /* background-position: right bottom; */
   background-position-y: 185px;
-  background-attachment: fixed;
   background-size: 25%;
   background-position: right bottom;
   padding-top: 10%;
   flex-grow: 1;
   height: 100vh;
+  background-attachment: fixed;
```

Comment:
[Optional] The background image on this section of your portfolio is not in the right format and how it ought to be according to the design, please replace the styling you have on your container with this 👇 to make it in the proper design template 😄  , remember that making your portfolio in the right design will make it attractive for visitors and intending recruiters:

>  .container {
       position : relative;
       background: url(Img/illustration-header-mobile.png);
       background-repeat: no-repeat;
        background-position: right bottom;
        padding-top: 10%;
        flex-grow: 1;
         height: 90vh;
                       }